src/content/en/tools/lighthouse/audits/time-to-interactive.md

```diff
@@ -2,33 +2,69 @@ project_path: /web/tools/_project.yaml
 book_path: /web/tools/_book.yaml
 description: Reference documentation for the "Time to Interactive" Lighthouse audit.
 
-{# wf_updated_on: 2018-07-23 #}
-{# wf_published_on: 2016-10-05 #}
-{# wf_blink_components: N/A #}
+{# wf_updated_on: 2018-08-16 #}
+{# wf_published_on: 2018-08-16 #}
+{# wf_blink_components: Platform>DevTools #}
 
-# Time to Interactive  {: .page-title }
+# Time to Interactive {: .page-title }
 
 ## Overview {: #overview }
 
-Page load is a key aspect of how a user perceives the performance of your
-page. See [Measure Performance with the RAIL Method](/web/fundamentals/performance/rail) for more information.
+<blockquote>
+  <p>
+    Load is not a single moment in time — it’s an experience that no one metric can fully capture.
+    There are multiple moments during the load experience that can affect whether a user perceives
+    it as "fast" or "slow".
+  </p>
+  <p>--- <a class="external" href="https://w3c.github.io/paint-timing/">Paint Timing spec</a></p>
+</blockquote>
 
-This audit identifies the time at which a page appears to be ready enough that
-a user can interact with it.
+The Time to Interactive (TTI) metric measures how long it takes a page to become interactive.
+"Interactive" is defined as the point where:
+
+* The page has displayed useful content, which is measured with [First Meaningful Paint][FMP].
+* Event handlers are registered for most visible page elements.
```

Comment:
This was the goal, but since we don't actually inspect event handlers at all I'm wondering if this is how we should frame it.

Maybe we should replace this bullet point with something about "most of the pages resources have loaded and finished executing" or something which is the real thing it does.

I'm surprised that https://developers.google.com/web/fundamentals/performance/user-centric-performance-metrics#time_to_interactive was the only thing I could really find defining it elsewhere :/